packages/web/src/components/createCell.tsx

```diff
@@ -122,10 +122,20 @@ export type CellSuccessProps<
  */
 export type DataObject = { [key: string]: unknown }
 
+type BeforeQueryHook<
```

Comment:
I think this needs to be exported:

```suggestion
export type BeforeQueryHook<
```